package.json

```diff
@@ -17,6 +16,242 @@
   },
   "homepage": "https://github.com/lewsmit2/cruise-ships#readme",
   "devDependencies": {
+    "eslint": "^8.30.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-import": "^2.26.0",
     "jest": "^29.3.1"
-  }
+  },
+  "dependencies": {
```

Comment:
Are all these dependencies actually needed? Theres an awful lot of extra packages in here!